src/Offer/Commands/DeleteOrganizer.php

```diff
@@ -0,0 +1,21 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Offer\Commands;
+
+final class DeleteOrganizer extends AbstractCommand
+{
+    private string $organizerId;
+
+    public function __construct(string $itemId, string $organizerId)
+    {
+        parent::__construct($itemId);
+        $this->organizerId = $organizerId;
+    }
+
+    public function getOrganizerId(): string
```

Comment:
Does `DeleteOrganizer` even need the organizer id? It could just delete the current organizer relationship?

Reply:
I think the derived event `OrganizerDeleted` would still expose the method `getOrganizerId`? In that case, it's an easy change because the aggregate can add the id of the current organizer.

Reply:
Events may always contain more info than strictly necessary to record the change.

Reply:
As discussed through Slack this will be handled in a dedicated PR that will also add an API call to remove an organizer on event/place/organizer without having to pass an organizer id.